docs/configuration.asciidoc

```diff
@@ -33,18 +33,20 @@ public class Startup
 }
 ----
 
-With this you can use any configuration source that you configured on the `IConfiguration` instance that you passed to the APM Agent. You can find the key of each configuration below in the IConfiguration key column.
+With this you can use any configuration source that you configured on the `IConfiguration` instance that you passed to the APM Agent.
+You can find the key of each configuration option
+in the `IConfiguration or Web.config key` column of the corresponding option's description.
 
 NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
 
 [float]
-[[sample-config]]
+[[configuration-on-asp-net-core-sample-config]]
```

Comment:
In the future, we want to avoid changing topic IDs. This breaks external links to the documentation. If a link _has_ to be changed, we want to add the old slug to a redirects file to catch any old links.

We should also take care to not have super long ids like this one. It makes it difficult to reference them.

Reply:
@bmorelli25 Do you think it would preferable to change the old link (this one) back to `sample-config` and [for the new one](https://github.com/elastic/apm-agent-dotnet/pull/503/files/5fadd946daff7f09456068a766103f0d026c4142#diff-01c1a79a5a1e1b67003a1885365f2cf5R104) use something like `asp-net-sample-config`?

Reply:
Yes, I'm working on a PR right now

Reply:
https://github.com/elastic/apm-agent-dotnet/pull/511